website/static/js/share/results.js

```diff
@@ -1,126 +1,252 @@
+                contributorViews :
+                m('span', [
+                    contributorViews.slice(0, 7),
+                    m('br'),
+                    m('a', {onclick: function(){result.showAllContrib = result.showAllContrib ? false : true;}}, 'See All')
+                ])
+        );
+
+    }
+};
+
+var Contributor = {
+    view: function(ctrl, params) {
+        var contributor = params.contributor;
+        var index = params.index;
+        var vm = params.vm;
+        return m('span', [
+            m('span', index !== 0 ? ' · ' : ''),
+            m('a', {
+                onclick: function() {
+                    utils.updateFilter(vm, 'match:contributors.familyName:' + contributor.familyName, true);
+                    utils.updateFilter(vm, 'match:contributors.givenName:' + contributor.givenName, true);
+                }
+            }, contributor.name)
+        ]);
+    }
+};
+
+var Subjects = {
+    view: function(ctrl, params){
+        var result = params.result;
+        var subjectViews = $.map(result.subjects || [], function(subject, i) {
+            return m.component(Subject, $.extend({subject: subject}, params));
+        });
+        if (result.showAllsubjects || (result.subjects || []).length <= 5) {
+            return m('span', subjectViews);
+        }
+        return m('span', [
+            subjectViews.slice(0, 5),
+            m('br'),
+            m('div', m('a', {onclick: function() {result.showAllsubjects = result.showAllsubjects ? false : true;}},'See All'))
+        ]);
+
+    }
+};
+
+var Subject = {
+    view: function(ctrl, params) {
+        var subject = params.subject;
+        var vm = params.vm;
+        return m('span', m('.badge.pointer', {onclick: function(){
+                utils.updateFilter(vm, 'match:subjects:"' + subject + '"', true);
+            }}, subject.length < 50 ? subject : subject.substring(0, 47) + '...'),
+            ' ');
+    }
+};
+
+var Footer = {
+    controller: function(params) {
+        var self = this;
+        this.result = params.result;
+        this.cleanResult = m.prop(null);
+        this.loadRawNorm = function() {
+            utils.loadRawNormalized(this.result).then(
+                function(cleanData) {
+                    self.cleanResult(cleanData);
+                }, function(error) {
+                    self.cleanResult(error);
+                }
+            );
+        };
+    },
+    view: function(ctrl, params) {
+        var result = params.result;
+        var vm = params.vm;
+        return m('div', [
+            m('span',
+                'Released on ' + new $osf.FormattableDate(result.providerUpdatedDateTime).local,
+                vm.rawNormedLoaded() ?  m('span', [
+                    m('span', {style: {'margin-right': '5px', 'margin-left': '5px'}}, ' | '),
+                    m('a', {
+                        onclick: function() {
+                            ctrl.showRawNormed = ctrl.showRawNormed ? false : true;
```

Comment:
This can be simplified to

```js
ctrl.showRawNormed = !ctrl.showRawNormed;
```

...although you might need to make `showRawNormed` an `m.prop`.